front_end/panels/ai_chat/core/__tests__/AgentNodesSanitize.test.ts

```
@@ -0,0 +1,80 @@
+// Copyright 2025 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import { createAgentNode } from '../AgentNodes.js';
+import type { AgentState } from '../State.js';
+import { ChatMessageEntity } from '../../models/ChatTypes.js';
```

ESLint

[error] `../../models/ChatTypes.js` import should occur before import of `../AgentNodes.js`

(import/order)